### PR TITLE
Logo for organization

### PR DESCRIPTION
**Pull request by jsopakar**

### CODE REVIEW

File: app/views/organizations/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,13 @@
-<h1><%= @organization.name %></h1>
+
+
```
Comment (jsopakar) [inline app/views/organizations/show.html.erb]: 2 empty rows?

File: spec/usermanual/teachers.html.erb [hunk 2]
```diff
@@ -93,6 +93,7 @@ require 'cgi' # for escapeHTML
     <ul>
       <li><b>Name</b> of the organization that will be seen in the app</li>
       <li><b>Description</b> about your organization, seen in this organization's page</li>
+      <li><b>Logo</b> for your organization, seen in this organization's page (you can change this and other fields above later)</li>
```
Comment (jsopakar) [inline spec/usermanual/teachers.html.erb]: Was there some requirements for the logo, like minimum size? If not, good, if there is, would be good to tell them here.

Comment (Termanty) [inline spec/usermanual/teachers.html.erb]: Any size is ok. Paperclip will automagic way scale picture 
to right size.

Comment (jamox) [inline spec/usermanual/teachers.html.erb]: Though we don't really want to upscale the image...

Comment (Termanty) [inline spec/usermanual/teachers.html.erb]: Current size of the logo is 100*100. I think most of the logos are bigger than that. Should we still restrict acceptance of images at least that 100*100 size?

Comment (jamox) [inline spec/usermanual/teachers.html.erb]: At max maybe add note of the recommended size to the teachers.html.erb...

**Comment by jamox** [comment]
LGTM

**Comment by jsopakar** [comment]
Closed, implemented in new pr, #321

**Pull request closed without merging**